src/components/ui/BaseCard.vue

```diff
@@ -128,4 +132,9 @@ function formatDate(date: Date): string {
 function toggleDropDown(dropDownClicked: boolean) {
   isDropDownClicked.value = dropDownClicked;
 }
+
+function todoChecked() {
```

Comment:
Please make sure to use the proper function name and use actions or use `on` prefix if it's triggered for a specific event. For example: `checkTodo` or `onTodoChecked`